lib/ansible/plugins/terminal/qnos.py

```diff
@@ -0,0 +1,96 @@
+#
+# (c) 2016 Red Hat Inc.
```

Comment:
Please update the year in the header for all the files

Reply:
Update the year to 2019